subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/local/LocalReservationClient.kt

```diff
@@ -46,14 +46,14 @@ class LocalReservationClient(
 
         val devicesChannel: Channel<WorkerDevice> = reservations
             .iterateInParallel { _, reservation ->
-                val deploymentName = generateDeploymentName()
-                reservationDeployments.send(deploymentName)
+                val fakeDeploymentName = "local-stub"
```

Comment:
"Deployment" part triggers me. What do you think about renaming `reservationDeployments` to something else?
I can't propose smthg but we can add todo or whatever.

Reply:
Renaming is not enough.
I'll try to hide the deployment state and don't expose it to clients at all.
I want to try this approach: https://youtu.be/a3agLJQ6vt8?t=1750